python/graphstorm/trainer/lp_trainer.py

```diff
@@ -323,16 +323,16 @@ def eval(self, model, data, val_loader, test_loader,
                                           edge_mask=edge_mask_for_gnn_embeddings,
                                           task_tracker=self.task_tracker)
         sys_tracker.check('compute embeddings')
-        val_scores = lp_mini_batch_predict(model, emb, val_loader, self.device) \
+        val_rankings = lp_mini_batch_predict(model, emb, val_loader, self.device) \
```

Comment:
Shall we still keep the val_scores? As in the future, lp_mini_batch_predict may return other scores than ranking scores.

Reply:
yes, make sense. Reverted back to "*_scores"